### PR TITLE
Vulcan: Problems List: Scaffold UI

### PR DESCRIPTION
**Pull request by ianrohde**
## Issue

First stage in implementing Problems UI. I got to the point where real data will help to proceed faster.

[Figma](https://www.figma.com/file/QVLUEpDVoyiMM4MXEGq1L9/iFixit---Problems?type=design&node-id=1-122&mode=design&t=ayzLTnsYXY4NW2PN-0)

Dev Vercel preview: https://react-commerce-gkp6ufvql-ifixit.vercel.app/Troubleshooting/Dryer

<details>
<summary>Problems List schema</summary>

```
export type ProblemProps = {
   problemTitle: string;
   deviceTitle: string;
   description: string;
   altText: string;
   imageSrcLg?: string;
   imageSrcThumb?: string;
   problemTypeIcon?: FaIconProps;
   badges?: string[];
};
```
```
{
   imageSrcLg: undefined,
   imageSrcThumb:
      'https://guide-images.cdn.ifixit.com/igi/GEQaIapMrXAqEjAS.thumbnail',
   problemTypeIcon: faVolumeXmark,
   problemTitle:
      'GE Refrigerator Model Number GSS25WSTASS (2008) Temperature Inconsistencies Involving the Fresh Food Compartment',
   deviceTitle:
      'GE Refrigerator Model Number GSS25WSTASS (2008) Sub-Model Number GSS25W',
   description:
      'Fluctuating temperatures in the refrigerator or freezer compartments, often due to a faulty thermostat or temperature sensor. The following text is for testing purposes only. Lets see how well this long text wraps with a cap on 4 lines.',
   altText: 'GE Refrigerator image',
},
```

</details>

## CR

- Uses static JSON to get a general schema shape (`mockProblems` const)
- Behind previous set feature flag: `TROUBLESHOOTING_COLLECTIONS_ENABLED`
	- enabled on dev: `.env.development`
	- disabled on prod: `.env.production`
- Problem List skipped for now. Not sure if this should be alongside the wiki Troubleshooting page in `/templates/`.
- Added UI knowns to Problems list problems like line limits and mobile layout switching

Closes https://github.com/iFixit/ifixit/issues/51178

### CODE REVIEW

File: frontend/app/(defaultLayout)/Troubleshooting/[device]/components/troubleshootingProblems.tsx [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+         <Stack as="main" spacing={3}>
+            <Box className="header">
+               <Heading color="gray.800" fontSize="30px" fontWeight="medium">
+                  Most Common {title} Troubleshooting Problems
+               </Heading>
+               <Text mt={2}>
+                  When your {title} runs into issues, it can be a source of
+                  frustration and inconvenience. Never fear &mdash; we&apos;ve
+                  compiled a comprehensive guide to diagnose and fix the most
+                  common problems that can plague your refrigerator.{' '}
```
Comment (jarstelfox) [inline frontend/app/(defaultLayout)/Troubleshooting/[device]/components/troubleshootingProblems.tsx:103]: Is this space needed?

```suggestion
                  common problems that can plague your refrigerator.
```

**Review by jarstelfox — DISMISSED** [review]
CR 🌵 but small deploy_block 🌵 I think it may pay off to update the generateMetadata function with your mocked data. At least that way you won't forget to set metadata once we have actual data

**Comment by ianrohde** [comment]
> At least that way you won't forget to set metadata once we have actual data

I'll check with Max on what he wants here. Thanks, that could have slipped through 👍

**Comment by ianrohde** [comment]
I opened a separate issue for metadata, [here](https://github.com/iFixit/ifixit/issues/51218).

We're going to have a fair amount of work between this merging that the project going live, so I don't want to hold up progress elsewhere. Again, thank for raising Jarred 🍻 

un_deploy_block 🟢

**Review by jarstelfox — APPROVED** [review]
CR 🌵

**Comment by erinemay** [comment]
QA 🌵 
- Flag still works. Can't see on prod preview. Can on dev preview. 
- There's some static stuff on the page now! 
image
- And some dynamic stuff. Cool!
image
image

**Pull request merged**